lib/components/narrative/default/default-itinerary.js

```diff
@@ -109,22 +111,25 @@ const ITINERARY_ATTRIBUTES = [
     render: (itinerary, options) => {
       const formattedStartTime = itinerary.startTime + options.offset
       const formattedEndTime = itinerary.endTime + options.offset
-      if (options.isSelected) {
+
+      if (itinerary.extraStartTimes) {
+        const extraStartTimes = Array.from(itinerary.extraStartTimes)
+        return (
+          <FormattedMessage
+            id='common.time.multiDepartureTimes'
+            values={{
+              startTime: formattedStartTime,
+              secondStartTime: extraStartTimes?.[0] + options.offset || 0,
+              thirdStartTime: extraStartTimes?.[1] + options.offset || 0
+            }}
+          />
+        ) 
+      }
         if (options.selection === 'ARRIVALTIME') {
-          return <FormattedTime value={formattedStartTime} />
-        } else {
           return <FormattedTime value={formattedEndTime} />
+        } else {
+          return <FormattedTime value={formattedStartTime} />
         }
```

Comment:
Fix indent.

Reply:
Fixed in 7969cc8c08590f6daf18c60716df01360bcd547b